shared/actions/wallets.js

```diff
@@ -215,11 +221,17 @@ const loadAccounts = (
     | ConfigGen.LoggedInPayload
 ) =>
   !actionHasError(action) &&
-  RPCStellarTypes.localGetWalletAccountsLocalRpcPromise().then(res =>
-    WalletsGen.createAccountsReceived({
+  RPCStellarTypes.localGetWalletAccountsLocalRpcPromise().then(res => {
+    // load the display currency of each wallet, now that we have the IDs
+    action.type === WalletsGen.loadAccounts &&
```

Comment:
This doesn't do anything (`map`s but doesn't do anything with the result). You could make a saga for `accountsReceived` that spawns those `loadDisplayCurrency` actions.

Reply:
But it does call `createLoadDisplayCurrency` - is there something else it needs to do?


VM2363:1 console.trace

  | (anonymous) | @ | VM2363:1
-- | -- | -- | --
  | createLoadDisplayCurrency | @ | wallets-gen.js:458
  | action.type._wallets_gen__WEBPACK_IMPORTED_MODULE_5__.loadAccounts.res.map.account | @ | wallets.js:204
  | _constants_types_rpc_stellar_gen__WEBPACK_IMPORTED_MODULE_2__.localGetWalletAccountsLocalRpcPromise.then.res | @ | wallets.js:204
  | _tickCallback | @ | internal/process/next_tick.js:68
  | Promise.then (async) |   |  
  | loadAccounts | @ | wallets.js:202
  | runCallEffect | @ | proc.js:523
  | runEffect | @ | proc.js:445

Reply:
I double checked and I'm certain it worked, but I changed it like you suggest.

Reply:
I don't think it could have, `actionToPromise` will `put` one action returned from the promise. Before, this was like
```
res.map(a => createLoadDisplayCurrency(...))
return createAccountsReceived(...)
```
so the `accountsReceived` will be dispatched but the mapped `loadDisplayCurrencies` don't do anything.